velox/substrait/tests/Substrait2VeloxPlanConversionTest.cpp

```diff
@@ -149,11 +169,11 @@ class Substrait2VeloxPlanConversionTest
         absolutePaths.emplace_back(fmt::format("{}{}", tempPath, path));
       }
 
-      std::vector<u_int64_t> starts = planConverter->getStarts();
-      std::vector<u_int64_t> lengths = planConverter->getLengths();
+      std::vector<u_int64_t> starts = splitStats->starts_;
```

Comment:
This test is hard to read. Please, consider refactoring as in https://github.com/facebookincubator/velox/pull/1686